### PR TITLE
Extract xaml header and closer generation to a separate class

### PR DESCRIPTION
**Pull request by georgii-borovinskikh-sonarsource**
It is going to be reused by Simple and Rich layout generators.

NOTE: no new code, just moved the bits to a separate class

### CODE REVIEW

File: src/Education/XamlGenerator/XamlGeneratorHelper.cs [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Xml;
+using SonarLint.VisualStudio.Rules;
+
+namespace SonarLint.VisualStudio.Education.XamlGenerator
+{
+    internal class XamlGeneratorHelper
```
Comment (ugras-ergun-sonarsource) [inline src/Education/XamlGenerator/XamlGeneratorHelper.cs:26]: Since this is now separated it needs it's own tests

Comment (ugras-ergun-sonarsource) [reply src/Education/XamlGenerator/XamlGeneratorHelper.cs:26]: Also it needs an interface

**Review by ugras-ergun-sonarsource — COMMENTED** [review]
I think new class should have tests and an interface.

File: src/Education/XamlGenerator/XamlGeneratorHelper.cs (hunk 1 above)
Comment (georgii-borovinskikh-sonarsource) [reply src/Education/XamlGenerator/XamlGeneratorHelper.cs:26]: I agree with adding test (although I didn't find any for the Simple layout generator, which is why I didn't add any), but I don't think adding an interface here would add any value. It's not going to be used in any way, not even in tests

**Review by ugras-ergun-sonarsource — APPROVED** [review]

**Pull request merged**